frontend/src/components/ui/popover.tsx

```diff
@@ -16,10 +16,18 @@ const PopoverContent = React.forwardRef<
   React.ElementRef<typeof PopoverPrimitive.Content>,
   React.ComponentPropsWithoutRef<typeof PopoverPrimitive.Content> & {
     portal?: boolean;
+    scrollable?: boolean;
   }
 >(
   (
-    { className, align = "center", sideOffset = 4, portal = true, ...props },
+    {
+      className,
+      align = "center",
+      sideOffset = 4,
+      portal = true,
+      scrollable = false,
```

Comment:
this one is intentionally default false since not sure all the types of popovers we have